config.py

```diff
@@ -102,6 +103,7 @@ class Test(Config):
     SECRET_KEY = "KEY"
 
     FEATURE_FLAGS_NEW_SUPPLIER_FLOW = enabled_since('2016-11-29')
+    FEATURE_FLAGS_DIRECT_AWARD_PROJECTS = True
```

Comment:
You should keep this as the `enabled_since` thing we use elsewhere to remain consistent. It should still pass the boolean check you do later.

Reply:
Any "truthy" value will turn the feature flag on - so *anything* other than `False` or `None` I think will work.

We do the `enabled_since` thing so we can see on the status pages for different environments which features are on and since when (e.g. see https://www.preview.marketplace.team/_status etc.)

Reply:
```
"flags": {
"FEATURE_FLAGS_TRANSACTION_ISOLATION": "2015-08-27"
}
```

Reply:
@samuelhwilliams @TheDoubleK - enabled_since always passed in False when running tests. Never evaluated the actual date.